src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/runtime/aarch64/AARCH64Frame.java

```diff
@@ -327,29 +327,22 @@ private Frame senderForEntryFrame(AARCH64RegisterMap map) {
     return fr;
   }
 
+  //------------------------------------------------------------------------------
+  // frame::adjust_unextended_sp
```

Comment:
This is just repeating the comment below.

Reply:
Thanks.